tests/contrib/botocore/test.py

```diff
@@ -266,6 +266,74 @@ def test_dynamodb_put_get(self):
         assert span.service == "test-botocore-tracing.dynamodb"
         assert span.resource == "botocore.parsers.parse"
 
+        span = spans[2]
+        assert span.get_tag("aws.operation") == "PutItem"
+        # Since the dynamodb_primary_key_names_for_tables isn't configured, we
+        # cannot create span pointers for this item.
+        assert not span._links
+
+    @pytest.mark.skipif(
+        PYTHON_VERSION_INFO < (3, 8),
+        reason="Skipping for older py versions whose latest supported moto versions don't have the right dynamodb api",
+    )
+    @mock_dynamodb
+    def test_dynamodb_put_get_with_table_primary_key_mapping(self):
```

Comment:
Can we also have another test for an error that may arise from this? Seems to be that we should be erroring if a user configures the feature wrong or we have too many primary keys. Just want to ensure error functionality works.

Reply:
i'll try it, but hopefully i've written it to fail, log, and not have span pointers when things are misconfigured

Reply:
yup. i was able to make a test that exercises the "fail safe" behavior!

Reply:
except now i seem to have lost push permissions for the repo. or something.

Reply:
ok, pushed!

Reply:
okay, LGTM, thanks for adding that!